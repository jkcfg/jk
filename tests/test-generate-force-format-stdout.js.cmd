jk generate --format=yaml --stdout ./generate-force-format.js
