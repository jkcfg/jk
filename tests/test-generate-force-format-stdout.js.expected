- item1: JSON
- item2: JSON
- item3: JSON
---
item1: JSONStream
---
item2: JSONStream
---
item3: JSONStream
---
- item1: YAML
- item2: YAML
- item3: YAML
---
item1: YAMLStream
---
item2: YAMLStream
---
item3: YAMLStream
